mne/io/base.py

```diff
@@ -831,13 +848,80 @@ def get_data(self, picks=None, start=0, stop=None,
         .. versionadded:: 0.14.0
         """
         picks = _picks_to_idx(self.info, picks, 'all', exclude=())
+
+        # Convert into the specified unit
+        ch_factors = np.ones(len(picks))
+
+        si_units = _handle_default('si_units')
+        si_units_splitted = {key: si_units[key].split('/') for key in si_units}
+        prefixes = {'': 1e0, 'd': 1e1, 'c': 1e2, 'm': 1e3, 'µ': 1e6, 'u': 1e6,
+                    'n': 1e9, 'p': 1e12, 'f': 1e15}
```

Comment:
`prefixes` looks like something general that may be reused in the future, so I would put it to `defaults.py`, and get them with `_handle_default('unit_prefixes'`).